pkg/neorpc/errors.go

```diff
@@ -138,6 +138,18 @@ const (
 	ErrExecutionFailedCode = -608
 )
 
+var (
+	// ErrCompatGeneric is an error returned by nodes not compliant with the neo-project/proposals#156
+	// (NeoGo pre-0.102.0 and all known C# versions).
+	// It can be returned for any call and doesn't have any specific meaning.
+	// Deprecated: to be removed after all nodes adopt new error standard.
+	ErrCompatGeneric = NewErrorWithCode(-100, "RPC error")
+	// ErrCompatNoOpenedWallet is an error code returned by nodes not compliant with the neo-project/proposals#156
+	// (all known C# versions, NeoGo never used this code). It can be returned for wallet-related operations.
+	// Deprecated: to be removed after all nodes adopt new error standard.
+	ErrCompatNoOpenedWallet = NewErrorWithCode(-400, "No opened wallet")
```

Comment:
Follow the https://github.com/nspcc-dev/.github/blob/53c43ecf6bf083f04b1d2d4b0f54012b8fe1cc79/git.md?plain=1#L63 for commit messages, please.

Reply:
Done, cut it in 80-chars length.